data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1408,7 +1419,11 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean warehouse: Port
             purpose = pgettext("atlanteans_building", "Serves as a base for overseas colonization and trade. Also stores your soldiers, wares and tools."),
             -- TRANSLATORS: Note helptext for an atlantean warehouse: Port
-            note = pgettext("atlanteans_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it.")
+            note = pgettext("atlanteans_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it."),
+            -- TRANSLATORS: Note lore for an atlantean production site: Port
+            lore = pgettext("atlanteans_building", "This is the building where expeditions will be equipped to boldly go and find new shores."),
+            -- TRANSLATORS: Note lore_autor for an atlantean production site: Port
+            lore_author = pgettext("atlanteans_building", "Ostur dedicating a new Port")
```

Comment:
```suggestion
            lore_author = pgettext("atlanteans_building", "Ostur, the designer of the new Expedition ship class, during the opening of a new Port")
```

Reply:
Will take that, not sure if such long authors are wise.